lib/puppet/application/cert.rb

```diff
@@ -132,7 +148,10 @@ def help
   List outstanding certificate requests. If '--all' is specified, signed
   certificates are also listed, prefixed by '+', and revoked or invalid
   certificates are prefixed by '-' (the verification outcome is printed
-  in parenthesis).
+  in parenthesis). If '--human-readable' or '-H' is specified listed
+  certificates are formatted in a way to improve human scan-ability. If
+  '--machine-readable' or '-m' is specified output is formatted in a
+  line-wise format optimal for scripted consumption.
```

Comment:
I feel like I've seen that in other places, maybe the term was line-oriented?

Either way, I've updated it to be similar to your suggestion so at a minimum we won't have to change it if we change the format (: